web/js/containers/timeline/timeline.js

```diff
@@ -217,6 +222,19 @@ class Timeline extends React.Component {
       }
     }
 
+    const subdailyAdded = hasSubdailyLayers && !prevProps.hasSubdailyLayers;
+    const subdailyRemoved = !hasSubdailyLayers && prevProps.hasSubdailyLayers;
+    const subdailyInterval = customInterval > 3 || interval > 3;
+
+    if (subdailyRemoved && subdailyInterval) {
+      changeCustomInterval();
+      selectInterval(1, TIME_SCALE_TO_NUMBER.day, false);
+    }
+
+    if (subdailyAdded && !customSelected) {
+      changeCustomInterval(10, TIME_SCALE_TO_NUMBER.minute);
```

Comment:
This also happens [here](https://github.com/nasa-gibs/worldview/blob/main/web/js/components/timeline/custom-interval-selector/custom-interval-selector.js#L49) which means that, at least on desktop, we dispatch this same action twice when a subdaily layer is added.  I'm guessing we can take the code out of that file and dispatching it here will work on both desktop and mobile?

Reply:
I've updated the CustomIntervalSelector component to remove the duplicate logic for changing intervals. The actions were no longer necessary in this component since it is always happening in the timeline parent component now. There should be no more duplicate actions.